week1-section3/exercise 5/Main.cpp

```diff
@@ -0,0 +1,46 @@
+/*Exercise 5 - Arrays
+Create an array of whatever size you want. Let the user input as many numbers as they want and populate the array with them. If they enter any invalid input, output the whole array of numbers and the sum of them all and the total numbers they input.
+
+If the user exceeds the size of your array, tell them about it and then output the information.
+*/
+
+#include <iostream>
+
+
+
+
+int main(){
+
+	int numbers[10];
```

Comment:
Remember that there is no need to declare all your variables before your logic. Declare your variables as close to their definition and as close to their use as possible.